app/retail/templates/ui_inventory.html

```diff
@@ -0,0 +1,921 @@
+            </div>
+          </div>
+          <span class="closebtn">×</span>
+        </div>
+      </div>
+
+      <div class="container-fluid p-5">
+        <p class="h1">{% trans "Animations" %}</p>
+        <div class="row pt-5 mb-3 animatedParent">
+          <div class="col d-flex flex-column align-items-center animated fadeInLeft go">
+            <div class="mb-3 d-flex justify-content-center align-items-center">
+              <img width="128" style="transform: rotateZ(90deg)" src="{% static "v2/images/regulus_square.png" %}">
+            </div>
+          </div>
+          <div class="col d-flex flex-column align-items-center animated fadeInRight go">
+            <div class="mb-3 d-flex justify-content-center align-items-center">
+              <img width="128" style="transform: rotateZ(270deg)" src="{% static "v2/images/regulus_square.png" %}">
+            </div>
+          </div>
+        </div>
+
+        <p>{% trans "Other whole page animations" %}</p>
+        <p>include 'shared/rain.html' with class="purple"</p>
+        {% include 'shared/rain.html' with class="purple" %}
+
+        <button class="btn btn-gc-purple" onclick="startConfetti()">{% trans "Start Confetti" %}</button>
+      </div>
+
+      <div class="container-fluid p-5 bg-white">
+        <p class="h1">{% trans "Other Elements" %}</p>
+        <div class="card p-1">
+          <p class="font-title-lg">progress-bar</p>
+          <span class="progress-bar">
+            <span class="progress" style="width: 32%;"></span>
+          </span>
+        </div>
+        <div class="container-fluid p-5">
+          <p class="h2">{% trans "Quotes" %} | waitingRoomEntertainment()</p>
+          <p class="quote_author"></p>
+          <p class="quote_text"></p>
+        </div>
+      </div>
+
+      <div class="container-fluid p-5">
+        <p class="h1">{% trans "Header" %}</p>
+        <p class="m-3">include 'shared/minihero.html' with h1='shared/minihero.html'</p>
+        <div class="container-fluid header">
+          {% include 'shared/minihero.html' with h1='shared/minihero.html' %}
+        </div>
+        <p class="m-3">include 'shared/hero.html'</p>
+        <div class="container-fluid header">
+          {% include 'shared/hero.html' %}
+        </div>
+      </div>
+
+      <div class="container-fluid p-5">
+        <p class="h1">{% trans "Modal Dialog" %}</p>
+        <div class="white-light-bg modal">
+          <div class="col-12 closebtn">
+            <a rel="modal:close" href="javascript:void" class="close" aria-label="Close dialog">
+              <span aria-hidden="true">&times;</span>
+            </a>
+          </div>
+          <div class="col-12">
+            <h5 class="text-center font-title">{% trans "Modal Dialog" %}</h5>
+          </div>
+        </div>
+      </div>
+
+      <div class="container-fluid p-5 assets">
+        <p class="h1">{% trans "Vector Graphics" %}</p>
+        <img src="{% static "v2/images/quickstart-box.svg" %}"/>
```

Comment:
I am not saying we need to do it here... but we'll probably want to modify the vast majority of the asset handling and repetitive entries here to simply loop over assets  in `images/` and the associated subs / context data.  We'd drop the line count in this file down by quite a bit and we wouldn't have to manually add every new asset as they're added (I'd be willing to bet it won't be long until we're missing assets here).

Reply:
That would be very nice. I'm not familiar with Django, so I will dig into that today. 👍